### DISCUSSION TITLE
How to force-recreate the Dependencies Dashboard issue?

### DISCUSSION THREAD

**Discussion by dotansimha** [renovatebot/renovate#32161] [Request Help]
### How are you running Renovate?

A Mend.io-hosted app

### Please tell us more about your question or problem

We are running on the mend-hosted, with GitHub. We accidentally deleted the "Dependencies Dashboard" issue, and now the Mend developer portal shows an error: "Could not ensure issue". 

How this could be fixed? can we tell Renovate to recreate the issue? 

### Logs (if relevant)

<details><summary>Logs</summary>

```

WARN: Could not ensure issue
{
  "err": {
    "name": "HTTPError",
    "code": "ERR_NON_2XX_3XX_RESPONSE",
    "timings": {
      "start": 1730013931863,
      "socket": 1730013931863,
      "lookup": 1730013931863,
      "connect": 1730013931863,
      "secureConnect": 1730013931863,
      "upload": 1730013931864,
      "response": 1730013931930,
      "end": 1730013931931,
      "phases": {
        "wait": 0,
        "dns": 0,
        "tcp": 0,
        "tls": 0,
        "request": 1,
        "firstByte": 66,
        "download": 1,
        "total": 68
      }
    },
    "message": "Response code 410 (Gone)",
    "stack": "HTTPError: Response code 410 (Gone)\n    at Request.<anonymous> (/usr/local/renovate/node_modules/.pnpm/got@11.8.6/node_modules/got/dist/source/as-promise/index.js:118:42)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)",
    "options": {
      "headers": {
        "user-agent": "RenovateBot/38.120.1 (https://github.com/renovatebot/renovate)",
        "accept": "application/json, application/vnd.github.machine-man-preview+json",
        "authorization": "***********",
        "if-none-match": "W/\"b90e2fd1810e4b644e12235aea91cce052f740926f5d7c79c6c5a844ea460288\"",
        "if-modified-since": "Mon, 21 Oct 2024 11:17:24 GMT",
        "accept-encoding": "gzip, deflate, br"
      },
      "url": "https://api.github.com/repos/graphql-hive/platform/issues/476",
      "hostType": "github",
      "username": "",
      "password": "",
      "method": "GET",
      "http2": false
    },
```

</details>

**Comment by rarkins** [comment]
Please give longer logs including debug before this happened. 

Renovate maybe thinks it's closed and can be reopened.

**Reply by dotansimha** [reply]
Hi @rarkins , thanks. 

Please see all logs in a text file. The actual issue was deleted 

[renovate-log.txt](https://github.com/user-attachments/files/17534109/renovate-log.txt)

I think these are the relevant lines? 
```
DEBUG: findIssue(Dependency Dashboard)
DEBUG: Issues cache: synced
DEBUG: Found issue 476
DEBUG: GET https://api.github.com/repos/graphql-hive/platform/issues/476 = (code=ERR_NON_2XX_3XX_RESPONSE, statusCode=410 retryCount=0, duration=50)
DEBUG: Error getting issue
```

The issue no longer exists, and GitHub seems to show that: https://github.com/graphql-hive/platform/issues/476 (I guess Renovate bot should consider this issue status?)

**Reply by rarkins** [reply]
Yes I think we might want renovate to be aware of delete issues. Either way by filtering them out (being blind to them) or treating them as a new status different from open/closed

**Reply by dotansimha** [reply]
Yeah, that makes sense, thank you.

If I understand correctly, changing `dependencyDashboardTitle` in the configuration to a new "key" will make Renovate recreate it? 🤔  

https://github.com/renovatebot/renovate/blob/1224fde2b8ce72b59b98b9d268ea0d678da2f687/lib/workers/repository/dependency-dashboard.ts#L139-L142

**Reply by rarkins** [reply]
That might work, yes

**Reply by dotansimha** [reply]
Yeah, the workaround works :) https://github.com/graphql-hive/platform/issues/5753 
Thanks @rarkins . Should I report an issue for this? or is there an existing one?

**Reply by rarkins** [reply]
There's no existing issue for this. I will mark this discussion for follow up and create an issue later

**Reply by HonkingGoose** [reply]
I found two issues that seem related to this problem:

- #21072
- #19563

I'll let you decide if you want to update a current issue, or make a new one for this problem. 😉

**Reply by rarkins** [reply]
I think they're different

**Reply by RahulGautamSingh** [reply]
- https://github.com/renovatebot/renovate/issues/32267
